src/main/client/src/components/app/table/react-table/ReactTable.js

```diff
@@ -558,8 +560,13 @@ class ReactTable extends Component<Props, State> {
             setRowFilter,
             activeAdminTool,
             activeTable,
+            setActivePage,
+            activePage,
         } = this.props;
 
+        let layerPage = activePage.page.find(o => o.layerId === layerFeatures.id);
+        let currentPage = layerPage ? layerPage.page : 0;
```

Comment:
Nämä voisi muuttaa olemaan const eikä let, sillä arvoja ei muuteta.
```suggestion
        const layerPage = activePage.page.find(o => o.layerId === layerFeatures.id);
        const currentPage = layerPage ? layerPage.page : 0;
```